doc/release-notes/5.1-release-notes.md

```diff
@@ -0,0 +1,78 @@
+# Dataverse 5.1
+
+This release brings new features, enhancements, and bug fixes to Dataverse. Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+## Release Highlights
+
+### Large File Upload for Installations Using AWS S3
+
+The added support for multipart upload through the API (Issue #6763) will allow files larger than 5 GB to be uploaded to Dataverse when an installation is running on AWS S3. Previously, only non-AWS S3 storage configurations would allow uploads larger than 5 GB.
```

Comment:
This is both API and UI! :-) (and DVUploader which uses the API) But only for direct uploads to S3. 

Also, not sure how much goes in the release notes but it might be good to note that:
- part size is configurable:
The new multipart functionality is used by default when direct upload files are > 1 GB. This value is configurable per store - see the Guides table of S3 settings. (Range is from 5 MB to 5GB per part if you want to list that here)
- Administrators should note that charges may be incurred for storage reserved for multipart uploads that are not completed or cancelled. Administrators may want to do periodic manual or automated checks for open multipart uploads.
- While multipart uploads can support much larger files, and can have advantages in terms of robust transfer and speed, they are more complex than single part direct uploads. Administrators should consider taking advantage of the options to limit use of multipart uploads to specific users (by using multiple stores and configuring access to stores with high file size limits to specific Dataverses or Datasets).

It's been long enough that I've forgotten how much documentation is out but I'll plan to create some additional info - perhaps for the guides or just wiki/faq/debug ideas or even some demo videos.

Reply:
Thanks @qqmyers - will address later today.

Reply:
@qqmyers - I addressed much of this in https://github.com/IQSS/dataverse/pull/7283/commits/f0a0d0b9f31a8315a5043cc3f9f0a7b872af64a9 ... If there's some better docs we can add pointers later.